src/builder.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{
+    config::Config,
+    value::Value,
+    source::Source,
+    error,
+    error::ConfigError,
+    path::Expression
+};
+use std::collections::HashMap;
+
+/**
```

Comment:
If this is not a matter of taste, then according to [Rust Reference](https://doc.rust-lang.org/reference/comments.html) doc comments are line comments that start with `///` or block comments that use `/** ... */` syntax (double asterisk at the beginning).

> Line doc comments beginning with exactly three slashes (///), and block doc comments (/** ... */), both inner doc comments, are interpreted as a special syntax for doc attributes. 

I did not know about it until VSCode suggested this and it actually works, I tested with `cargo doc`. Thought this is convinient for multiline doc comments.

Reply:
Yes, the reference is the syntax. In [the official styleguide](https://doc.rust-lang.org/1.0.0/style/style/comments.html) they use `//` instead of `/*` (andn `///` instead of `/**`).

Reply:
Let it be. Changed.